subsys/infinispan/src/main/resources/infinispan.xml

```diff
@@ -1,75 +1,70 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<infinispan
-    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:schemaLocation="urn:infinispan:config:9.1 http://www.infinispan.org/schemas/infinispan-config-9.1.xsd"
-    xmlns="urn:infinispan:config:9.1">
+<infinispan xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xsi:schemaLocation="urn:infinispan:config:9.1 http://www.infinispan.org/schemas/infinispan-config-9.1.xsd"
+            xmlns="urn:infinispan:config:9.1">
 
   <cache-container default-cache="local" name="IndyCacheManager" shutdown-hook="DEFAULT" statistics="true">
     <local-cache-configuration name="local-template" statistics="true">
-      <eviction strategy="LRU" size="200000" type="COUNT"/>
+      <!-- eviction is DEPRECATED, use memory element instead -->
+      <memory>
+        <!--
+          Limits the cache to this by the amount of entries in the cache.
+          If choose memory or off-heap, make sure the key/value objects implements Serializable
+        -->
+        <object size="200000"/>
+      </memory>
     </local-cache-configuration>
 
     <local-cache name="local" configuration="local-template"/>
 
-    <local-cache name="koji-maven-version-metadata" configuration="local-template">
-      <eviction strategy="LRU" size="200000" type="COUNT"/>
-    </local-cache>
+    <local-cache name="koji-maven-version-metadata" configuration="local-template"/>
 
     <local-cache name="folo-in-progress" configuration="local-template">
-      <eviction size="200000" type="COUNT" strategy="LRU"/>
       <indexing index="LOCAL">
         <property name="hibernate.search.model_mapping">org.commonjava.indy.folo.data.FoloCacheProducer</property>
         <property name="default.directory_provider">ram</property>
-        <!-- <property name="hibernate.search.default.indexBase">${indy.data}/folo/search</property> -->
       </indexing>
     </local-cache>
 
     <local-cache name="folo-sealed" configuration="local-template">
-      <eviction size="1000" type="COUNT" strategy="LRU"/>
+      <memory>
+        <object size="1000"/>
+      </memory>
       <persistence passivation="true">
         <file-store shared="false" preload="false" fetch-state="true" path="${indy.data}/folo"/>
       </persistence>
     </local-cache>
 
-    <local-cache name="content-index" configuration="local-template">
-      <eviction strategy="LRU" size="200000" type="COUNT"/>
-    </local-cache>
+    <local-cache name="content-metadata" configuration="local-template"/>
```

Comment:
This cache config was missing. Could have caused some unpredicted problems. Add it.